pkg/yurthub/storage/disk/storage.go

```diff
@@ -57,9 +57,9 @@ func NewDiskStorage(dir string) (storage.Store, error) {
 		baseDir:          dir,
 	}
 
-	err := ds.Recover("")
-	if err != nil {
+	if err := ds.Recover(""); err != nil {
 		klog.Errorf("could not recover local storage, %v, and skip the error", err)
+		return nil, err
```

Comment:
as error info said: `and skip the error`, we should only dump the error info instead of exiting yurthub, because of some temp file can not recover should not prevent starting up yurthub.